unlock-app/src/services/currencyLookupService.js

```diff
@@ -0,0 +1,59 @@
+import axios from 'axios'
+
+const URL = require('url')
+
+/**
+ *  Provides currency lookup abstraction for down stream consumers.
+ */
+export default class CurrencyLookupService {
+  constructor(uri) {
+    /* eslint-disable node/no-deprecated-api */
```

Comment:
I am worried by this. Please add a comment to explain what's going on?

Reply:
Our lint rules currently flag this as an deprecated method for node 11. As we aren't using this version it is added noise and is blocking the build. I'll add a note with the understand that this is should be resolved in the future and this removed